xcs-ui/xcs-ui-common/src/main/resources/FlamingoThemes/XCS-Blue.xml

```diff
@@ -946,9 +946,8 @@
 @gadget-shadow:none;
 /* livetable */
 @th-livetable-border:2px solid #999;
-@th-livetable-bg:#FFFFFF;
 @td-livetable-border:1px solid #ccc;
-@td-livetable-padd-btm:10px;
+@td-livetable-padd:5px 5px 5px 0;
```

Comment:
why this value? the padding-left is 0 so the text is very close to the left margin.